app/build.gradle

```diff
@@ -191,6 +192,7 @@ android {
     defaultConfig {
         minSdkVersion 9
         targetSdkVersion 25
+        multiDexEnabled true
```

Comment:
pretty big change to move to multiDex. This is 100% needed for tihs work, right?